pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotTenantRestletResource.java

```diff
@@ -690,9 +685,52 @@ public SuccessResponse deleteTenant(
   @ApiOperation(value = "Rebalances all the tables that are part of the tenant")
   public TenantRebalanceResult rebalance(
       @ApiParam(value = "Name of the tenant whose table are to be rebalanced", required = true)
-      @PathParam("tenantName") String tenantName, @ApiParam(required = true) TenantRebalanceConfig config) {
+      @PathParam("tenantName") String tenantName,
+      @ApiParam(value = "Number of table rebalance jobs allowed to run at the same time", required = true, example =
+          "1")
+      @QueryParam("degreeOfParallelism") Integer degreeOfParallelism,
+      @ApiParam(value =
+          "Comma separated list of tables with type that should be included in this tenant rebalance"
+              + " job. Leaving blank defaults to include all tables from the tenant. Example: table1_REALTIME, "
+              + "table2_REALTIME",
+          example = "")
+      @QueryParam("includeTables") String allowTables,
+      @ApiParam(value =
+          "Comma separated list of tables with type that would be excluded in this tenant rebalance"
+              + " job. These tables will be removed from includeTables (that said, if a table appears in both list, "
+              + "it will be excluded). Example: table1_REALTIME, table2_REALTIME",
+          example = "")
+      @QueryParam("excludeTables") String blockTables,
```

Comment:
can you rename the variables as well to `includeTables` and `excludeTables` to make them consistent?